src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -111,26 +110,48 @@ static daos_handle_t          eq_list[MAX_EQ];
 static uint16_t               eq_count_max;
 static uint16_t               eq_count;
 static uint16_t               eq_idx;
+
+/* the number of EQs for aio contexts. (aio_eq_count + eq_count) <= eq_count_max */
 static uint16_t               aio_eq_count;
 
-struct aio_ctx_eq_ev {
-	uint64_t      queued_op_count;
-	daos_handle_t eq;
-	daos_event_t  ev;
-	struct iocb  *piocb;
+struct aio_eq {
+	daos_handle_t    eq;
+	pthread_mutex_t  lock;
+	_Atomic uint64_t n_op_queued;
+	_Atomic uint64_t n_op_done;
+};
+
+/* list of EQs dedicated for aio contexts. */
+static struct aio_eq          aio_eq_list[MAX_EQ];
+/* the accumulated iodepth for aio context. */
+static long int               depth_sum_accu;
+
+struct aio_ev {
+	daos_event_t    ev;
+	struct iocb    *piocb;
+	struct aio_ctx *ctx;
 };
 
 struct aio_ctx {
+	/* the real io_context_t used in libaio */
 	io_context_t          ctx;
+	/* the depth of contex set by io_setup */
```

Comment:
codespell
```suggestion
	/* the depth of context set by io_setup */
```

Reply:
Thank you! I will update it in next commit.

Reply:
Typo is fixed now. Thank you!